ivy/functional/backends/jax/elementwise.py

```diff
@@ -12,7 +12,13 @@
 from . import backend_version
 
 
-def abs(x: Union[float, JaxArray], /, *, out: Optional[JaxArray] = None) -> JaxArray:
+def abs(
+    x: Union[float, JaxArray],
+    /,
+    *,
+    where: Union[bool, JaxArray] = None,
+    out: Optional[JaxArray] = None,
+) -> JaxArray:
```

Comment:
We should be making use of the `where` argument inside the implementation as the boolean mask applied